main.js

```diff
@@ -0,0 +1,17 @@
+const addButton = document.getElementById('add');
+const itemsContainer = document.getElementById('itemsContainer');
+const inputField = document.getElementById('inputField');
+
+addButton.addEventListener('click', () => {
+  const paragraph = document.createElement('p');
+  paragraph.classList.add('p-style');
+  paragraph.innerText = inputField.value;
+  itemsContainer.appendChild(paragraph);
+  inputField.value = '';
+  paragraph.addEventListener('click', () => {
+    paragraph.style.textDecoration = 'line-through';
+  });
+  paragraph.addEventListener('dblclick', () => {
+    itemsContainer.removeChild(paragraph);
```

Comment:
- [ ] Nice job for adding this method to remove the to-do when the user double click on the text. But in the requirements of this project, we are asked to use a function to **clear all completed** buttons using  `Array.filter()` method. Please let's try it out!.